srcpkgs/simavr/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'simavr'
+pkgname=simavr
+version=1.6
+revision=1
+build_style=gnu-makefile
+make_build_args="RELEASE=1"
+make_build_target="build-simavr build-parts"
+make_use_env=1
+hostmakedepends="pkg-config"
+makedepends="elfutils-devel libfreeglut-devel"
+short_desc="A lean, mean and hackable AVR simulator"
+maintainer="Martijn van Buul <martijn.van.buul@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/buserror/${pkgname}"
+distfiles="https://github.com/buserror/${pkgname}/archive/v${version}.tar.gz"
+checksum=a55ad04d055eef5656c49f78bc089968b059c6efb6a831618b8d7e67a840936d
+
+CFLAGS+="-fPIC"
+do_install() {
+	make DESTDIR=${DESTDIR}/usr PREFIX=/usr RELEASE=1 install
+
+	# Package installs directory containing a single headerfile which is
+	# intended to be used by avr-gcc for AVR-targets into the host's
+	# include directory.  Move it to /usr/avr/include instead. This also
+	# prevents it from being moved into the 'devel' subpackage, which is a
+	# good thing as it is required at runtime.
+
+	mkdir -p ${DESTDIR}/usr/avr/include
+	mv ${DESTDIR}/usr/include/simavr/avr ${DESTDIR}/usr/avr/include/simavr
```

Comment:
I don't think /usr/include is a good location; this file is intended to be used with avr-gcc exclusively (not the system-wide gcc). Installing it in the default location in /usr/include means mixing header files intended for avr-gcc (and avr-libc) with files intended for the host's gcc (and MUSL or glibc). This is the same reason why avr-libc 's headerfiles are located in a different location than /usr/include. And rightly so. 

The only reason I chose /usr/avr is because that's where avr-libc installs its header files. I don't think that's the best location either, but that isn't really my decision to make. It seemed to makes sense to follow avr-libc's lead though: The headerfiles intended for the host are installed in a subdirectory of the default include directory for the host's gcc (in other words: in /usr/include), the headerfiles intended for the target are installed in a subdirectory of the default include directory for avr-gcc (in other words: /usr/avr/include).

Reply:
I'm not opposed to moving it to (a subdirectory of) /usr/share, but I feel like avr-libc (and indirectly: avr-gcc) should follow suit in that case